### PR TITLE
Add Tron address interface

### LINKED ISSUE #221: Missing JNI generated TronAddress class

### PR DESCRIPTION
**Pull request by alejandro-isaza**
Fix #221

### CODE REVIEW

File: swift/Sources/TronAddress.swift [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+// file LICENSE at the root of the source code distribution tree.
+//
+// This is a GENERATED FILE, changes made here WILL BE LOST.
+//
+
+import Foundation
+
+    public final class TronAddress {
+
+  public
```
Comment (hewigovens) [inline swift/Sources/TronAddress.swift]: The formatting in this file sucks

**Comment by hewigovens** [comment]
could you please also remove `swift/Sources/Addresses/TronAddress.swift`? Let's use wallet core's implementation

File: swift/Sources/TronAddress.swift (hunk 1 above)
Comment (alejandro-isaza) [reply swift/Sources/TronAddress.swift]: That was the clang-format commit hook 🤦‍♂️

Comment (hewigovens) [reply swift/Sources/TronAddress.swift]: lol

**Comment by alejandro-isaza** [comment]
Done

**Pull request merged**